src/trace.ml

```diff
@@ -561,6 +561,32 @@ module Make_commands (Backend : Backend_intf.S) = struct
     { Decode_opts.output_config; decode_opts; print_events }
   ;;
 
+  let open_url_flags =
```

Comment:
I think we should allow configuration of this through envvars in `env_vars.ml`, rather than as commandline parameters. These choices tend to be sticky, so it's nicer to not have users have to specify them over and over.

I'd say a `MAGIC_TRACE_WEB_UI_URL` that can be any string, and which defaults to `https://magic-trace.org`, would be sufficient here.

Reply:
Once I implement the Env_vars configuration, should I use the commandline flag to indicate whether the user wants to display the ui, or should it always be displayed?

Reply:
I think for now, we should have a separate flag for if the trace processor shell should be used, if available. Maybe in the future we can make the selection automatic using some heuristic based on the size of the trace file (and whether we expect a browser to safely open it), but let's hold off on that bit for now.